Header header

int8 ROBOT = 1
int8 JS = 2
int8 VS = 3

int8 ACTION_START = 1
int8 ACTION_PAUSE = 2
int8 ACTION_END = 3
int8 ACTION_OPEN_MENU = 4 #?
int8 ACTION_SHOW_MENU = 5
int8 ACTION_CLICK_MENU = 6 #?
int8 ACTION_SEND_JS_DATA = 7 #?
int8 ACTION_LOAD_DEFAULT_BG = 8
int8 ACTION_CLOSE = 9
int8 ACTION_LOAD_WELCOME_BG = 10
int8 ACTION_LOAD_IMAGE = 11
int8 ACTION_USER_CHECK_IN = 12
int8 ACTION_SET_DEFAULT_USERS = 13
int8 ACTION_RELOAD_SESSION_START = 14
int8 ACTION_RELOAD_OPEN_MENU = 15

int8 STATE_READY = 101
int8 STATE_END = 102
int8 STATE_GAME_SELECTED = 103
int8 STATE_USER_START = 104
int8 STATE_USER_CHECKED_IN = 105
int8 STATE_CONNECTED = 106

#topleft bottomleft topcenter bottomcenter topright bottomright
int8 MENU_TL = 1
int8 MENU_BL = 2
int8 MENU_TC = 3
int8 MENU_BC = 4
int8 MENU_TR = 5
int8 MENU_BR = 6
